Development/src/components/JSONViewer.js

```diff
@@ -1,19 +1,41 @@
+import React from 'react';
 import {
     Card,
     CardContent,
     Collapse,
     FormControlLabel,
     Switch,
 } from '@material-ui/core';
+import { useTheme } from '@material-ui/styles';
 import get from 'lodash/get';
-import React from 'react';
 import ReactJson from 'react-json-view';
 
+// Base16 dark theme color palette
+const rjv_dark = {
```

Comment:
What does 'rjv' stand for? Where do these values come from?

Reply:
rjv stands for react-json-viewer. Base 16 is a theming standard for code https://github.com/chriskempson/base16 
I'll leave a comment and maybe change the name